spyder/plugins/application/container.py

```diff
@@ -284,73 +279,99 @@ def _check_updates_ready(self):
-
-                msg = header + content + footer
+        elif update_available:
+            # Update using our installers
+            if (
+                not sys.platform.startswith("linux")
+                or parse(latest_release) >= parse("6.0.0")
+            ):
+                box.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+                box.setDefaultButton(QMessageBox.Yes)
+
+                msg = (
+                    header +
+                    _("Would you like to automatically download "
+                      "and install it? <br><br>")
+                )
+
+                if not is_pynsist() and not running_in_mac_app():
+                    msg += _("Clicking Yes will download and install the "
+                             "latest version using Spyder's installer, leaving"
+                             " your existing installation untouched.<br><br>")
+
+                msg += _("For more information, visit our "
+                         f"<a href=\"{url_i}\">installation guide</a>.")
+
                 box.setText(msg)
-                box.set_check_visible(True)
                 box.exec_()
-                if self.application_update_status:
-                    if box.result() == QMessageBox.Yes:
-                        self.application_update_status.start_installation(
-                            latest_release=latest_release)
-                    elif box.result() == QMessageBox.No:
-                        self.application_update_status.set_status_pending(
-                            latest_release=latest_release)
-                check_updates = box.is_checked()
-            elif feedback:
-                msg = _("Spyder is up to date.")
+                if box.result() == QMessageBox.Yes:
+                    self.application_update_status.start_installation(
+                        latest_release=latest_release)
+
+            # Manual update
+            if (
+                not box.result()  # The installer dialog was skipped
+                or (
+                    box.result() == QMessageBox.No
+                    and not is_pynsist()
+                    and not running_in_mac_app()
+                )
+            ):
+                # Update-at-startup checkbox visible only if manual update
+                # is first message box
+                box.set_check_visible(not box.result())
+                box.setStandardButtons(QMessageBox.Ok)
+                box.setDefaultButton(QMessageBox.Ok)
+
+                msg = _("")
+                if not box.result():
+                    msg += header
+
+                terminal = "terminal"
+                if sys.platform == "win32":
+                    if is_anaconda():
+                        terminal = "Anaconda prompt"
+                    else:
+                        terminal = "cmd prompt"
+                msg += _(f"Run the following commands in the {terminal} "
+                         "to update manually:<br>")
```

Comment:
```suggestion
                         "to update manually:<br><br>")
```

Add additional blank line.